src/main/java/ru/hogwarts/school/service/FacultyService.java

```diff
@@ -27,17 +31,25 @@ public Faculty createFaculty(Faculty faculty) {
 
     public Faculty findFaculty(long id) {
         logger.debug("findFaculty is running");
-        return facultyRepository.findById(id).get();
+        return facultyRepository.findById(id).orElseThrow(BadRequestException::new);
     }
 
     public Faculty editFaculty(Faculty faculty) {
         logger.debug("editFaculty is running");
-        return facultyRepository.save(faculty);
+        Optional<Faculty> test = facultyRepository.findById(faculty.getId());
+        if(test.isPresent()){
+            facultyRepository.save(faculty);
+        }else throw new BadRequestException("no faculty with this ID");
```

Comment:
```suggestion
        } else {
            throw new BadRequestException("no faculty with this ID");
        }
```
фигурные скобки всегда писать нужно